lib/config/ldapconfiguration.go

```diff
@@ -7,10 +7,10 @@
 package config
 
 type LDAPConfiguration struct {
-	LDAPAddress            string      `xml:"address,omitempty" json:"addresd"`
-	LDAPBindDn             string      `xml:"bindDn,omitempty" json:"bindDn"`
-	LDAPTLSMode            LDAPTLSMode `xml:"tlsMode,omitempty" json:"tlsMode"`
-	LDAPInsecureSkipVerify bool        `xml:"insecureSkipVerify,omitempty" json:"insecureSkipVerify" default:"false"`
+	LDAPAddress            string        `xml:"address,omitempty" json:"addresd"`
```

Comment:
So, given we are now inside an LDAP struct we can skip the LDAP prefix here as well. And capitalize DN in BindDN.